VGMPlay/dbus.c

```diff
@@ -22,6 +22,7 @@ They weren't lying when they said that using libdbus directly signs you up for s
 #include <dbus/dbus.h>
 #include "chips/mamedef.h"          // for UINT8
 #include "mmkeys.h"
+#include <inttypes.h>
```

Comment:
This won't work on VC6

Reply:
This code is never going to compile under VC6 in the first place. It's Linux only.